akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -838,6 +838,12 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
         // now again on stream 1
         sendDataAndExpectOnNet(entity1DataOut, 1, "jklm")
 
+        // send two data bits first but only pull and expect later
```

Comment:
Another test case bit I added here after I noticed that this worked but broke during the refactoring.